### PR TITLE
Add func to Delete a chain

### PR DESCRIPTION
**Pull request by sbezverk**
Signed-off-by: Serguei Bezverkhi <sbezverk@cisco.com>

In some cases deleting a specific chain is very desirable, adding api call and test for `DelChain`

### CODE REVIEW

**Comment by sbezverk** [comment]
@stapelberg Please take a look when you have a chance. Thanks so much..

File: chain.go [hunk 1]
```diff
@@ -114,6 +114,24 @@ func (cc *Conn) AddChain(c *Chain) *Chain {
 	return c
 }
 
+// DelChain deletes the specified Chain. See also
+// https://wiki.nftables.org/wiki-nftables/index.php/Configuring_chains#Adding_base_chains
```
Comment (stapelberg) [inline chain.go]: Update this link to https://wiki.nftables.org/wiki-nftables/index.php/Configuring_chains#Deleting_chains

File: chain.go [hunk 2]
```diff
@@ -114,6 +114,24 @@ func (cc *Conn) AddChain(c *Chain) *Chain {
...
+func (cc *Conn) DelChain(c *Chain) {
+
```
Comment (stapelberg) [inline chain.go]: Remove this blank line

File: chain.go (hunk 1 above)
Comment (sbezverk) [reply chain.go]: done

File: chain.go (hunk 2 above)
Comment (sbezverk) [reply chain.go]: done

**Pull request merged**